engine/src/main/java/org/terasology/rendering/opengl/FrameBuffersManager.java

```diff
@@ -90,6 +95,8 @@ public FrameBuffersManager() {
     Also instructs the PostProcessor and the GraphicState instances to fetch the FBOs they require.
      */
     public void initialize() {
+        fboManagerSubscribers = Lists.newArrayList();
```

Comment:
Could be a set I imagine, as there probably is no need for ordering the subscribers. On the other hand subscribers shouldn't be allowed to subscribe more than once and therefore receive the same notification more than once.